bin/sdss_access_curl_mangacube_dr14

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+from sdss_access import CurlAccess, AccessError
```

Comment:
I know we have this for the rsync examples, but maybe we can consolidate these examples into a single `sdss_access_curl_mangacube` file and just separate the different versions with comment blocks?

Reply:
yeah, its a bit of a mess, so I agree that cleanup is a good idea.

Reply:
however these examples are meant to run from the commandline for testing, so if they were combined it would confusing which tests were successful, especially since some are public and some are authentication.

Perhaps we could just pick a few good ones and delete the redundant ones?

Reply:
That's a good point on the command line use.  Ok, let's pick a few good ones and remove the other ones.

Reply:
Agreed